File: javascript/gemini/p110.js
const { spawnSync } = require('child_process');

const username = process.argv[2];

if (!username) {
    console.log("Usage: node script.js <username>");
    process.exit(1);
}

const child = spawnSync('id', [username], { stdio: 'inherit' });

console.log(`Exit status: ${child.status}`);

File: python/gemini/p110.py
import subprocess
import sys

if len(sys.argv) < 2:
    print("Usage: python script.py <username>")
    sys.exit(1)

username = sys.argv[1]

# run() returns a CompletedProcess instance
result = subprocess.run(["id", username])

print(f"Exit status: {result.returncode}")